xwiki-pro-macros-api/src/main/resources/css/tabmacro.css

```diff
@@ -0,0 +1,3 @@
+body.content.preference-underlining-only-inline-links #xwikicontent div.xwikitabmacro ul.nav.nav-tabs li a {
```

Comment:
I think the this selector is overly specific. 

I think this should suffice `div.xwikitabmacro ul.nav.nav-tabs li a`. Its possible to have xwiki syntax in places other than the xwiki content.